docs/permissions.md

```diff
@@ -372,6 +372,13 @@ Accept: application/vnd.api+json
             "codes": {
                 "jane": "Yohyoo8BHahh1lie"
             }
+        },
+        "cozyMetadata": {
+            "doctypeVersion": 1,
+            "metadataVersion": 1,
+            "createdAt": "2019-05-14T11:01:41.687737935+02:00",
+            "createdByApp": "drive",
```

Comment:
I think it should not be possible to change the `createdAt` and `createdByApp` once the permissions was created.

Reply:
I do not mind if it is technically possible (there is no need to add a protection if it's not planned), but no one should have to modify it in a normal situation

Reply:
I kept the option to modify it, but removed the documentation lines.